src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -254,6 +266,8 @@ class ContentOpenWith extends PureComponent<Props, State> {
                 } else {
                     windowRef.opener = null;
                     windowRef.location = url;
+                    onExecute(this.executeId);
+                    this.executeId = null;
```

Comment:
Should there be a wrapping handler to avoid the duplication of nulling out this property?

Reply:
yep, will add that in.